libs/jsonforms-components/src/lib/Controls/Inputs/InputEnumRadios.tsx

```diff
@@ -12,10 +12,15 @@ import { checkFieldValidity } from '../../util/stringUtils';
 type RadioGroupProp = EnumCellProps & WithClassname & TranslateProps & WithInputProps;
 
 export const RadioGroup = (props: RadioGroupProp): JSX.Element => {
-  const { data, className, id, enabled, schema, uischema, path, handleChange, options, config, label, t } = props;
+  const { data, id, enabled, schema, uischema, path, handleChange, options, config, label, t } = props;
   const enumData = schema?.enum || [];
   const appliedUiSchemaOptions = merge({}, config, props.uischema.options, options);
   const errorsFormInput = checkFieldValidity(props as ControlProps);
+
+  if (uischema?.options?.isStepperReview) {
+    return <div data-testid="input-enum-radios-review">{data}</div>;
```

Comment:
For these data test ids can you make them unique as there maybe key/ids collisions and it may cause issues with e2e tests

Reply:
That is a good idea. I will do that for sure.